guides-and-tutorials/sbtc/how-to-run-sbtc-signer.md

```diff
@@ -0,0 +1,531 @@
+# TODO(429): Add well-known seed nodes
+# TODO(715): Add well-known seed nodes
+seeds = [
+    "<protocol>:<ip>:<port>",
+    "provided:provided-ip:provided-port"
+]
+
+# The local network interface(s) and port(s) to listen on.
+#
+# You may specify multiple interfaces and ports by adding additional entries to
+# the list. Entries can be addressed by any of IPv4 address, IPv6 address or
+# hostname. Note that not all networks have IPv6 enabled, so it is recommended
+# to provide an IPv4 address as well.
+#
+# Specifying a port of `0` will cause the server to bind to a random port,
+# and an IP of `0.0.0.0` will cause the server to listen on all available
+# interfaces.
+#
+# Available protocols:
+# - tcp: Standard TCP socket connections.
+# - quick-v1: QUIC over UDP. This protocol is faster and uses less bandwidth,
+#       but may not be supported by all nodes' networks. Nodes will always
+#       attempt QUIC connections first, and fall back to TCP if it fails.
+#       If UDP is blocked on your network then you should not specify a QUIC
+#       listener (as it will never be reachable).
+#       More information: https://en.wikipedia.org/wiki/QUIC
+#
+# Format: ["<protocol>:<ip>[:port]", ...]
+# - If port is omitted then the default port 4122 will be used.
+# Default: ["tcp://0.0.0.0:4122", "quic-v1://0.0.0.0:4122"]
+# Required: false
+# Environment: SIGNER_SIGNER__P2P__LISTEN_ON
+#
+# Defined in the provided docker compose, do not uncomment unless you know
+# what you're doing.
+# -------------------------------------------------------------------------
+listen_on = ["tcp://0.0.0.0:4122", "quic-v1://0.0.0.0:4122"]
+
+# The publicly accessible network endpoint to advertise to other nodes.
+#
+# If this is not specified then the node will attempt to use other peers on the
+# network to determine its public endpoint. This is the recommended
+# configuration for most users.
+#
+# If your network uses an advanced configuration with separate inbound/outbound
+# addresses then you must specify this value with your inbound address and
+# configure port-forwarding as auto-discovery will report your outbound address.
+#
+# Format: ["<protocol>:<ip>:<port>", ...] (see `listen_on` for protocol options)
+# Default: <none>
+# Required: false
+# Environment: SIGNER_SIGNER__P2P__PUBLIC_ENDPOINTS
+public_endpoints = []
+```
+
+</details>
+
+### Network Protocol Details
+
+The signer operates on port 4122 by default and supports both TCP and QUIC protocols for peer communication. The signer will attempt QUIC connections first for improved performance, automatically falling back to TCP if QUIC is unavailable or blocked on the network.
+
+## 3. Set up your blocklist client
+
+The blocklist client provides address screening services for the signer node. It interfaces with external API services to perform risk analysis on Bitcoin addresses. Default implementation uses Chainalysis, but supports custom implementations.
```

Comment:
We'll want signers to use Chainalsys _sanctioned_ addresses API (https://www.chainalysis.com/free-cryptocurrency-sanctions-screening-tools/#:~:text=Cryptocurrency%20sanctions%20screening%20API,is%20on%20a%20sanctions%20list.) not the risk check.

On the linked page they can also request an API key.